p2p/peer.go

```diff
@@ -399,19 +399,23 @@ func createMConnection(
 			// because msgBytes is on socket receive buffer yet so reactor can read it concurrently
 			copied := make([]byte, len(msgBytes))
 			copy(copied, msgBytes)
-			select {
-			case ch <- &BufferedMsg{
-				ChID: chID,
-				Peer: p,
-				Msg:  copied}:
-			default:
-				// if the channel is full, we abandon this message
-				// Should check `config.Config.XxxBufSize`
-				p.Logger.Error("Lost the message since BaseReactor.recvMsgBuf is full",
-					"reactor", reactor,
-					"msgBytes.len", len(msgBytes), "msgBytes", fmt.Sprintf("%X", msgBytes))
-				p.metrics.NumAbandonedPeerMsgs.With(labels...).Add(1)
-			}
+			ch <- &BufferedMsg{ChID: chID, Peer: p, Msg: copied}
+			/*
+				select {
+				case ch <- &BufferedMsg{
+					ChID: chID,
+					Peer: p,
+					Msg:  copied}:
+				// Remove `default` case since shouldn't abandon message although blocking sending message into golang channel
+				//default:
+				//	// if the channel is full, we abandon this message
+				//	// Should check `config.Config.XxxBufSize`
+				//	p.Logger.Error("Lost the message since BaseReactor.recvMsgBuf is full",
+				//		"reactor", reactor,
+				//		"msgBytes.len", len(msgBytes), "msgBytes", fmt.Sprintf("%X", msgBytes))
+				//	p.metrics.NumAbandonedPeerMsgs.With(labels...).Add(1)
+				}
+			*/
```

Comment:
I think these can be removed, but is there any reason to leave it commented out?

Reply:
I want that nobody uses `select case and default`.

Reply:
I thought it'd be better to leave only a one-line comment such that "fixed to block and wait to avoid abandoned messages".